.github/workflows/seo.yml

```
@@ -11,55 +20,133 @@ on:
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git add website/sitemap.xml website/robots.txt website/seo.json website/site-data.json website/visitor-stats.json || true
+          git add website/sitemap.xml website/robots.txt website/seo.json website/site-data.json website/visitor-stats.json
           if git diff --staged --quiet; then
-            echo "No SEO artifact changes"
+            echo "No SEO artifact changes to commit"
           else
-            git commit -m "chore(seo): weekly site-data and sitemap refresh"
+            git commit -m "$(cat <<'EOF'
+chore(seo): refresh site-data, sitemap, and SEO manifest.
+
+Automated marketplace sync, visitor stats, and sitemap lastmod update.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)